sbi/samplers/mcmc/slice_numpy.py

```diff
@@ -514,8 +517,9 @@ def run_slice_np_vectorized(inits, seed):
     with tqdm_joblib(
         tqdm(
             range(num_batches),  # type: ignore
-            disable=not show_progress_bars,
-            desc=f"Running {num_chains} MCMC chains in {num_batches} batches.",
+            disable=not show_progress_bars or num_workers == 1,
```

Comment:
I don't understand this. Why would we not show a progressbar if `num_workers==1`?

Reply:
with multiple workers we suppress the individual MCMC progress bars and show this outer progress bar. when we have only a single worker we don't and show the actual MCMC progress bar instead.

Reply:
Makes sense, thanks!